src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Internal/Services/InternalConnectionsService.cs

```diff
@@ -348,6 +344,66 @@ public async Task<ValidationProblemInstance> RemovePackage(Guid fromId, Guid toI
         return null;
     }
 
+    public async Task<Result<IEnumerable<AccessPackageDto.Check>>> CheckPackage(Guid party, IEnumerable<string> packages, IEnumerable<Guid> packageIds = null, CancellationToken cancellationToken = default)
```

Comment:
Ikke egentlig behov for å duplisere DelegationCheck inn i internal da det ikke er noen funksjonell forskjell fra enduser endepunktet. Skaper bare dobbelt arbeid når vi evt. må endre på noe.